rules/linux/macos_clear_system_logs.yml

```diff
@@ -0,0 +1,25 @@
+title: Indicator Removal on Host - Clear Mac System Logs
+id: acf61bd8-d814-4272-81f0-a7a269aa69aa
+status: experimental
+description: Detects deletion of local audit logs
+author: remotephone, oscd.community
+date: 2020/10/11
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1070.002/T1070.002.md
+logsource:
+    product: macos
+    category: process_creation
+detection:
+    selection:
+      - ProcessName: 'rm' 
```

Comment:
could you please change processname to:

```
      - ProcessName: '/bin/rm'
```

or to the 

```
      - ProcessName|endswith: '/rm'
```

The thing is that process monitoring on unix-like operating systems based on syscalls, and syscall executed with the absolute path to the binary, and in your specific example you declared that it supposed to be equal to `rm` (it is not gonna fly). Here is the santa log for `rm` execution:

```
"process": {
      "args": [
        "rm",
        "-f",
        "/private/var/log/osquery/osquery-output.log"
      ],
      "executable": "/bin/rm",
    },
```